bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -66,6 +67,7 @@
 		<xs:attribute name="required" type="xs:boolean" />
 		<xs:attribute name="readOnly" type="xs:boolean" />
 		<xs:attribute name="multiple" type="xs:boolean" />
+		<xs:attribute name="unit" type="config-description:unitType" />
```

Comment:
hm, I thought we agreed to use a string attribute for this, so that we do not have to maintain the list of valid units with in the schema on the long run. What was the reason that you now implemented it as an enum?